rewrite-java/src/test/java/org/openrewrite/java/ChangeAnnotationAttributeNameTest.java

```diff
@@ -38,4 +38,104 @@ class A {}
             )
         );
     }
+
+    @Test
+    void doNotChangeIdenticalAssignment() {
+        rewriteRun(
+          spec -> spec.recipe(new ChangeAnnotationAttributeName("java.lang.SuppressWarnings", "value", "value")),
+          java(
+            """
+              @SuppressWarnings(value = {"foo", "bar"})
+              class A {}
+              """
+          )
+        );
+    }
+
+    @Test
+    void renameValueAttribute() {
+        rewriteRun(
+          spec -> spec.recipe(new ChangeAnnotationAttributeName("org.example.Foo", "value", "bar")),
+          java(
+            """
+              package org.example;
+              public @interface Foo {
+                  String value() default "";
+                  String bar() default "";
+              }
+              """
+          ),
+          java(
+            """
+              import org.example.Foo;
+                            
+              @Foo(/* some comment */ "1.0")
+              class A {}
+              """,
+            """
+              import org.example.Foo;
+                            
+              @Foo(/* some comment */ bar = "1.0")
+              class A {}
+              """
+          )
+        );
+    }
+
+    @Test
+    void expandImplicitValueAttribute() {
+        rewriteRun(
+          spec -> spec.recipe(new ChangeAnnotationAttributeName("java.lang.SuppressWarnings", "value", "value")),
+          java(
+            """
+              @SuppressWarnings({"foo", "bar"})
+              class A {}
+              """,
+            """
+              @SuppressWarnings(value = {"foo", "bar"})
+              class A {}
+              """
+          )
+        );
+    }
+
+    @Test
+    void rewriteEnumValue() {
```

Comment:
This test case may not be strictly necessary, it was helpful to me during development to get the value assignment case working.